ci/jobs/build_clickhouse.py

```diff
@@ -167,6 +167,8 @@ def main():
             targets = "clickhouse-bundle"
         elif build_type == BuildTypes.FUZZERS:
             targets = "fuzzers"
+        elif build_type == BuildTypes.AMD_DEBUG:
+            targets = "-k0 all"
```

Comment:
This is completely ignored because `amd_debug` is within the `BUILD_TYPE_TO_DEB_PACKAGE_TYPE` dict.

We can check in the log of this same PR that it didn't work: https://s3.amazonaws.com/clickhouse-test-reports/PRs/82167/3e25692c37fbf9e4fdd922cee43c3b538cfc61d7//build_amd_debug/job.log

```
> Start execution for [Build ClickHouse]
Run command: [ninja clickhouse-bundle]
```

If we want `AMD_DEBUG` to build all targets, we should move the if statement prior to the `if build_type in BUILD_TYPE_TO_DEB_PACKAGE_TYPE`

Reply:
Hi @pamarcos 

I don't know is we want to build on amd, but in any case please open a new issue referencing this (or reopen this one, as you prefer)